.env.example

```diff
@@ -17,3 +17,9 @@ EMAIL_HOST_USER= email_host_user
 EMAIL_HOST_PASSWORD = email_host_password
 
 EMAIL_PORT = EMAIL_PORT
+
+CLOUD_NAME = <your-cloud-name>
```

Comment:
@myrdstom Add these to new env values to the app.json file e.g 
`"CLOUD_NAME": {
        "required": true
      }` 
and also the heroku staging environment variables. This will enable the review app for this pull request to be created.

Reply:
Im going to need some clarity on this